server/parser.go

```diff
@@ -205,7 +207,7 @@ func (c *client) parse(buf []byte) error {
 				if err := c.processPub(c.trace, arg); err != nil {
 					return err
 				}
-				c.drop, c.as, c.state = OP_START, i+1, MSG_PAYLOAD
+				c.drop, c.as, c.state = 0, i+1, MSG_PAYLOAD
```

Comment:
Why would you not have OP_START here which is easier to read IMO?

Reply:
type safety; c.drop is int, OP_START is parserState (( that is categorical value )).

Reply:
Ah, gotcha. Thanks!